pages/queue.js

```diff
@@ -47,15 +49,22 @@ class Queue extends React.Component {
 
   render() {
     const { isFetching, hasQueue } = this.props
+
     if (isFetching) {
       return <Loading />
     }
     if (!isFetching && !hasQueue) {
       return <Error statusCode={404} />
     }
+    const locationText = this.props.queue.location || 'No location specified'
     return (
       <Layout>
         <Container fluid>
+          <h3>{this.props.queue.name}</h3>
+          <h5 className="text-muted">
```

Comment:
Can you add a bottom margin here? Maybe `mb-4`.

Reply:
```
         <Col
            xs={{ size: 12 }}
            md={{ size: 4 }}
            lg={{ size: 3 }}
            className="mb-3"
          >
            <h3>{this.props.queue.name}</h3>
            <h5 className="text-muted">
              <FontAwesomeIcon icon={faMapMarker} fixedWidth className="mr-2" />
              {locationText}
            </h5>
          </Col>
```
Would this be how I add the bottom margin?

Reply:
You can just add it directly to the `h5`